inputs/text.vue

```diff
@@ -110,7 +110,8 @@
       // every time the value of the input changes, update the store
       update(val) {
         if (this.isNumerical) {
-          val = parseFloat(val, 10);
```

Comment:
I 🔪 'd this `10` because I don't think `parseFloat` takes a 2nd arg(?)

Reply:
yuuup, that's `parseInt`